i18n/locale_conf.json

```
@@ -1 +1 @@
-{"translations": [ "de", "en", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
+{"translations": [ "de", "es","en", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
```

```suggestion
{"translations": [ "de", "en", "es", "fr", "ja", "ko", "zh-Hans", "zh-Hant", "zh-Hant-HK" ]}
```
Sort alphabetically.